CHANGELOG.rst

```diff
@@ -6,6 +6,207 @@ Changelog
 
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
+3.3.0 (2025-09-?)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+Upgrade procedure
+-----------------
+
+To upgrade to 3.3, please:
+
+* ⚠️ Ensure you upgrade to Open Forms 3.2.x before upgrading to the 3.3 release series.
+* ⚠️ Plan an upgrade window to address the warnings below.
+* ⚠️ An automatic Ogone-to-Wordline merchant migration requires unique merchant PSPIDs. This is
+  automatically verified *before* performing the upgrade, but ideally you should run the check script of
+  older patch versions in preparation.
+
+.. warning:: Schedule the upgrade for off-peak hours. Some of the database migrations
+   need to lock the entire table and/or can take a long time depending on the amount of
+   data. Some benchmarks on one million of rows in the submission variables table
+   (~ 120K submissions) showed a migration time of around 20 seconds, so anywhere
+   between 10 seconds - 5 minutes can be expected as a normal completion time depending
+   on your data and available resources for the database.
+
+.. warning::
+
+    In this release, we reworked the internal data type information. To ensure submitted
+    data of existing submissions is formatted correctly, submitted variables need to be
+    processed. Note that we include this script instead of a data migration, so it can be
+    run separately, as it can take up to an hour to complete the entire operation for large
+    environments.
+
+    .. code-block:: bash
+
+        # in the container via ``docker exec`` or ``kubectl exec``:
+        python /app/bin/fix_submission_value_variable_missing_fields.py
+
+.. warning::
+
+    For the email and confirmation templates, and the registration backends, we changed
+    the way that the data is generated. In case of key conflicts between static, component,
+    and user-defined variables, the static variables will take precedence. Previously, the
+    component and user-defined variables would override the static variables. Our validation
+    guards against the use of keys that are already present in the static variables, but this
+    does not cover old forms and newly-added static variables.
+
```

Comment:
This is missing the "Major features" section, see the other releases likes 3.2.0 and 3.1.0.


```suggestion
Major features
--------------

**💳 Worldline (Ogone replacement) support**

...

**🗺️ Map layers and advanced interactions**

...

**🚸 Children component with prefill**

...


**📈 Added application metrics**

...
```